### PR TITLE
Adding Seifart-2015-414 conceptlist

### PR DESCRIPTION
**Pull request by MuffinLinwist**
# Pull request checklist

- [x] add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
- [ ] checked whether the new concept(s) can be applied to existing lists with `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

  This is the requested concept list extracted from the [lexibank/seifartecheverriboran](https://github.com/lexibank/seifartecheverriboranl) dataset.

### CODE REVIEW

**Comment by Tarotis** [comment]
Thanks for adding this list.

@chrzyki Could we ask you to moderate this PR? I'd provide the first review, and @AnnikaTjuka already accepted to take over the second review.

@MuffinLinwist Please remember that when opening the PR, it is better not to tag anybody specifically. Somebody will moderate the PR (check for basic things) and then assign the reviewers. This way, there is less confusion about who will actually work on a PR, and the tasks will get distributed by the Concepticon editors. This also assures a better overview on the editor's site about who is already doing multiple reviews and who could help.

**Comment by MuffinLinwist** [comment]
> Thanks for adding this list.
> 
> @chrzyki Could we ask you to moderate this PR? I'd provide the first review, and @AnnikaTjuka already accepted to take over the second review.
> 
> @MuffinLinwist Please remember that when opening the PR, it is better not to tag anybody specifically. Somebody will moderate the PR (check for basic things) and then assign the reviewers. This way, there is less confusion about who will actually work on a PR, and the tasks will get distributed by the Concepticon editors. This also assures a better overview on the editor's site about who is already doing multiple reviews and who could help.

Thanks for the comment. I already modified the opening message :)

**Comment by chrzyki** [comment]
> Thanks for adding this list.
> 
> @chrzyki Could we ask you to moderate this PR? I'd provide the first review, and @AnnikaTjuka already accepted to take over the second review.
> 
> @MuffinLinwist Please remember that when opening the PR, it is better not to tag anybody specifically. Somebody will moderate the PR (check for basic things) and then assign the reviewers. This way, there is less confusion about who will actually work on a PR, and the tasks will get distributed by the Concepticon editors. This also assures a better overview on the editor's site about who is already doing multiple reviews and who could help.

Happy to moderate! Thank you @MuffinLinwist for preparing the list and @Tarotis for doing the first and @AnnikaTjuka the second review.

The repository for this concept list is https://github.com/lexibank/seifartecheverriboran, correct?

One small thing for now: We've got

```
Journal={LIAMES: Línguas Indígenas Americanas},
```

and

```
journal={LIAMES}
```

for stating naming this particular journal and I'd suggest the full (i.e. former) form.

Thanks again and I'll have a closer look after the review(s).

**Comment by MuffinLinwist** [comment]
> > Thanks for adding this list.
> > @chrzyki Could we ask you to moderate this PR? I'd provide the first review, and @AnnikaTjuka already accepted to take over the second review.
> > @MuffinLinwist Please remember that when opening the PR, it is better not to tag anybody specifically. Somebody will moderate the PR (check for basic things) and then assign the reviewers. This way, there is less confusion about who will actually work on a PR, and the tasks will get distributed by the Concepticon editors. This also assures a better overview on the editor's site about who is already doing multiple reviews and who could help.
> 
> Happy to moderate! Thank you @MuffinLinwist for preparing the list and @Tarotis for doing the first and @AnnikaTjuka the second review.
> 
> The repository for this concept list is https://github.com/lexibank/seifartecheverriboran, correct?
> 
> One small thing for now: We've got
> 
> ```
> Journal={LIAMES: Línguas Indígenas Americanas},
> ```
> 
> and
> 
> ```
> journal={LIAMES}
> ```
> 
> for stating naming this particular journal and I'd suggest the full (i.e. former) form.
> 
> Thanks again and I'll have a closer look after the review(s).

Thanks for noticing this! It's done :)

File: concepticondata/conceptlists.tsv
Comment (Tarotis) [inline concepticondata/conceptlists.tsv:433]: It is "Juan", not Juán"

Comment (Tarotis) [reply concepticondata/conceptlists.tsv:433]: "It also reassess the genealogical relationship of Bora-Muinane with Witoto, Ocaina, and Nonuya"

No, it does not.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:66]: We can probably map this to the narrower PATH

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:69]: I think we could map this to AXE

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:81]: We could think about adding RED DEER as a sub-concept of DEER. Please open an issue about this

See https://concepticon.clld.org/parameters/1936

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:84]: Is the comma in "cebus, monkey" correct?

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:118]: This should be GRASSLAND (https://concepticon.clld.org/parameters/2819) instead

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:132]: I don't think this fits

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:146]: There are many entries in Concepticon regarding "ear ornament". Please open an issue whether those should be mapped under "earring", or whether we should add a new concept "ear ornament".

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:216]: This is the most generic "eat" in the list, compared to "eat meat" and "eat fruit". If @AnnikaTjuka agrees, we could map this to EAT, but I'd like to have a second opinion on this

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:297]: We can unite this with "steel axe", since they present the same forms. See note in the original concept list

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:295]: Same as "river" (=49)

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:302]: This is CASSAVA

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:305]: We can map this under 

  | BECOME SICK
-- | --


[BECOME SICK](https://concepticon.clld.org/parameters/2327)

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:308]: Are we sure this is the verb?

Comment (Tarotis) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:308]: I can ask the authors about it to reassure

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:331]: Same as "open grassland", please unite concepts

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:339]: We can map this as PORT

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:387]: Rifle is generally unmapped. We could also consider this as new concept

**Review by Tarotis — CHANGES_REQUESTED** [review]
Thanks for adding this list. I have made a couple of suggestions for edits.

In some cases, I think we can merge concepts, since the authors provide the same form anyway, and even give comments like `(Note =49,311)` (as for RIVER and BIG RIVER). In two other cases, I will ask the authors for clarification.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:66]: It was like that on the automatic linking but I change it based on previous mappings.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:84]: It appears like this on the original `raw file` on the dataset.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:122]: So the way to go over this is to have just one single entry:

Seifart-2015-414-120	120	leaf,tree leaf	628	LEAF	153,154

?

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:132]: What do you mean? Annatto is achiote in fact.

Comment (Tarotis) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:132]: I thought that ACHIOTE would refer to the tree, not the condiment, but I see that there are many previous mappings. So you are right.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:122]: Yes

File: concepticondata/conceptlists.tsv
Comment (AnnikaTjuka) [inline concepticondata/conceptlists.tsv:433]: I think we usually list the languages in the `target/source language` column with a comma instead of "and". Also, change "reassess" to "reassesses".

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:178]: These cases are a bit strange and I noticed that some lists are mapped go! to GO and come! to COME. Since the verbs don't seem to appear anywhere else on the list, I'd map them, but could you open an issue where we discuss these cases?

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:197]: Not all glosses of "shelter" are mapped to HUT so I'd unmap here.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:216]: I like the idea that there is a special word for eating meat and not the other way round. We have to indicate if someone eats vegetarian. I'd map it for now.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:302]: But CASSAVA was mapped previously. Can we merge the two?

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:387]: We discussed this before but decided to leave it for now. https://github.com/concepticon/concepticon-data/issues/1234

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Seifart-2015-414.tsv:397]: Could someone do a study on EAT colexifications with different kinds of food? Or is there already one? I've never seen this distinction in a concept list before.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:302]: In past lists, this is unmapped. Should I open an issue about that?

**Review by AnnikaTjuka — COMMENTED** [review]
Thanks for adding the list, @MuffinLinwist! I added a few comments.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:308]: I think better to ask for this too.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:302]: It seems that the `cassava (bread)` is usually the unmapped one so I'd do the same in this list, i.e., map `bitter manioc (plant)` and unmap `cassava (bread)`.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:387]: I add this at the list of issues to be opened in Concepticon.

Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:387]: Then, I'll just unmap the gloss on this list.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:178]: All of them do appear in an infinitive form in the list. But I can open an issue for us to discuss the cases where they don't.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:397]: There is but just [Kaufman-1973-1028-775](https://concepticon.clld.org/values/Kaufman-1973-1028-775).

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:302]: I agree with Annika here

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:178]: there is "come" in line 227. But "go" does seem to be missing indeed

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:302]: Perhaps `cassava (bread)` here could be mapped as [407 MANIOC BREAD](https://concepticon.clld.org/parameters/407)?

Comment (Tarotis) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:302]: Good catch! Seems to fit perfectly

Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:302]: Ok, I kept them separately and mapped them accordingly.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv:178]: So, I'll map `go!` with [695 GO](https://concepticon.clld.org/parameters/695), leave the rest unmapped, and open an issue about these cases.

**Comment by MuffinLinwist** [comment]
@Tarotis, @AnnikaTjuka, and @chrzyki, thanks for your first round of observations. The new version of the PR addresses all of your remarks. We are only waiting on confirmation from the authors of the list for some of the doubts here.

**Comment by Tarotis** [comment]
Since we have merged some of the concepts, the new number of concepts is now at 410, not 414. Please fix all the files that refer to this number. I will then make the final review.

**Comment by MuffinLinwist** [comment]
@Tarotis, the changes are done for you to do the final review.

**Comment by Tarotis** [comment]
Thanks. I'll do the review once I have received the answers of the authors regarding the concepts. Usually they respond to me quite quickly

**Comment by chrzyki** [comment]
Thank you all for preparing this list! Current status is that @Tarotis is still waiting for feedback by the authors, correct? Once that is done, @AnnikaTjuka and I can have another look at the list?

**Review by AnnikaTjuka — APPROVED** [review]
I'll be on holiday next week, so I'm approving the PR now. All my suggestions were implemented.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv]: There's already a response on #1337 on this topic from @LinguList.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [reply concepticondata/conceptlists/Seifart-2015-414.tsv]: Mapping is correct

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (Tarotis) [reply concepticondata/conceptlists/Seifart-2015-414.tsv]: This should be mapped as the verb "BLOW (WITH MOUTH)" @MuffinLinwist

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv]: This was already responded in #1336. We should map this gloss as EARRING. With the last commit, this was implemented.

**Comment by MuffinLinwist** [comment]
@chrzyki and @Tarotis, with the answers from the authors provided by @Tarotis, this PR has all the requested changes implemented and the pending topics resolved. The status of the PR is still waiting for @Tarotis to check if all of his suggestions were implemented and the PR is ready to be merged.

**Review by Tarotis — APPROVED** [review]
Thanks! From my side, we are good to go.

File: concepticondata/conceptlists/Seifart-2015-414.tsv
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Seifart-2015-414.tsv]: This is now pending for approval on the PR #1339.

Comment (chrzyki) [reply concepticondata/conceptlists/Seifart-2015-414.tsv]: Thank you for mapping this, approved.

**Comment by MuffinLinwist** [comment]
I adapted the changes on the #1339 PR and mapped the `red deer` gloss here. Since, we have all the approvals and all the checks have passed, I'll go ahead and merge the PR.

**Pull request merged**
